File: AppBox3/Apps/Abbreviation/Model/AbbreviationFavorite+CoreDataProperties.m
//
//  AbbreviationFavorite+CoreDataProperties.m
//  AppBox3
//
//  Created by Byeong-Kwon Kwak on 1/26/17.
//  Copyright © 2017 ALLABOUTAPPS. All rights reserved.
//

#import "AbbreviationFavorite+CoreDataProperties.h"

@implementation AbbreviationFavorite (CoreDataProperties)

+ (NSFetchRequest<AbbreviationFavorite *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"AbbreviationFavorite_"];
}

@dynamic order;
@dynamic uniqueID;

@end

File: AppBox3/Apps/Abbreviation/AbbreviationFavorite_+CoreDataProperties.swift
//
//  AbbreviationFavorite_+CoreDataProperties.swift
//  
//
//  Created by BYEONG KWON KWAK on 10/2/24.
//
//

import Foundation
import CoreData


extension AbbreviationFavorite_ {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<AbbreviationFavorite_> {
        return NSFetchRequest<AbbreviationFavorite_>(entityName: "AbbreviationFavorite_")
    }

    @NSManaged public var order: String?
    @NSManaged public var uniqueID: String?

}
